Desktop/Regression_Test_Plan_Minor_Release.md

```diff
@@ -93,22 +93,22 @@ TestID | Test Case | Steps to reproduce |Expected Result |Result | Related Comme
+------------ | ------------- | -------------- | ----- | ------ | ------ | ----
+1 :robot: | Verify that you can create one folder  | 1. Go to local sync folder 2. Create a single folder 3. Wait for sync | The folder is visible on the server | :heavy_check_mark: | tst_syncing | :robot: oC10,<br> :robot: oCIS |
+2 | Verify that you can create one folder with long name  | 1. Go to local sync folder 2. Create a single folder with a long name (about 100 characters) 3. Wait for sync |The folder visible on the server | :heavy_check_mark: | tst_syncing | |
+3 :robot: | Verify that you can create one folder with special characters in the name | 1. Go to local sync folder 2. Create a single folder with a special character(s) in the name (e.g $%ñ&) 3. Wait for sync | The folder appears on the server | :heavy_check_mark: | tst_syncing | :robot: oC10,<br> :robot: oCIS |
+4 :robot: | Verify that you can sync many subfolders | 1. Go to local sync folder 2. Copy a folder with 5 empty subfolders and 5 folders containing files 3. Wait for sync | All 10 subfolders are visible on the server | :heavy_check_mark: | tst_syncing | :robot: oC10,<br> :robot: oCIS |
 5 | Verify that you can create multiples folders at once | 1. Create a folder with 400 subfolders outside the sync root, 2. Move that into the sync root, 3. Wait for sync  | All 400 folders appear on the server | :construction: | |
-6 :robot: | Verify that you can copy one folder | 1. Go to local sync folder 2. Create a single folder with some files in it 3. Copy and paste the folder 4. Wait for sync | Both copies appears on the server | :heavy_check_mark: | tst_syncing |
-7 :robot: | Verify that you can create a subfolder with long name | 1. Go to local sync folder, 2. Create a folder called "Folder1" 3. Create a subfolder called "LUsgzq!0k02sek+szBqrzN5=R#UJpWql&rwhnYVb~Gh!l!”  (optionally up to 220 characters), 4. This subfolder had a file called ilppng.PNG inside it 5. Wait for sync |The files are synced correctly| :heavy_check_mark: | tst_syncing |
-8 :robot: | Verify pre existing folders in local (Desktop client) are copied over to the server | 1. Turn off the Desktop client 2. Go to local sync folder 3. Create several folders inside the Desktop Client folder at several different levels, 4. Turn ON the Desktop Client  | Folders appear on the server | :heavy_check_mark: | tst_syncing |
-9 :robot: | Filenames that are rejected by the server are reported | 1. Go to local sync folder, 2. Create a file called `"a\\a"` (or another name not accepted by the server), 3. Wait for sync | The sync status reports an error, the file s not synced| :heavy_check_mark: | tst_syncing |
+6 :robot: | Verify that you can copy one folder | 1. Go to local sync folder 2. Create a single folder with some files in it 3. Copy and paste the folder 4. Wait for sync | Both copies appears on the server | :heavy_check_mark: | tst_syncing |  :robot: oC10|
```

Comment:
should be covered here https://github.com/owncloud/client/blob/0fbaf515aa67e3265a2309ed2033061af53f5e6e/test/gui/tst_syncing/test.feature#L190
```suggestion
6 :robot: | Verify that you can copy one folder | 1. Go to local sync folder 2. Create a single folder with some files in it 3. Copy and paste the folder 4. Wait for sync | Both copies appears on the server | :heavy_check_mark: | tst_syncing |  :robot: oC10,<br> :robot: oCIS |
```